ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -869,9 +959,9 @@ open class BottomCommandingController: UIViewController {
     }
 
     private class HeroItemBindingInfo: ItemBindingInfo {
-        let widthConstraint: NSLayoutConstraint
+        let widthConstraint: NSLayoutConstraint?
```

Comment:
Why is this optional now?

Reply:
Having it be non-optional meant we were applying the constraints to the overflow hero items, which was leading to constraint errors. Now we can pass in `nil`, and the `.compactMap` in `heroCommandWidthConstraints` will only return the non-nil constraints for the hero items showing in the tab bar.